src/provider/cloud-element-templates/properties/CustomProperties.js

```diff
@@ -138,6 +139,14 @@ function createCustomEntry(id, element, property) {
   }
 
   if (type === 'String') {
+    if (feel) {
```

Comment:
Can I test it with `npm run start:cloud-templates`? If not, please add a template which allows to see it in action.